spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -3,8 +3,21 @@
     given_there_is_an_edition
     when_i_go_to_edit_the_edition
     and_i_go_to_insert_an_attachment
+    and_i_select_a_file
     and_i_upload_a_file_attachment
     then_i_can_see_previews_of_the_attachment
+    and_i_can_see_a_link_to_the_attachment
+    and_i_see_the_timeline_entry
+  end
+
+  scenario do
+    given_there_is_an_edition_with_a_pre_release_document_type
+    when_i_visit_the_summary_page
+    and_i_go_to_change_an_attachment
+    and_i_go_to_upload_an_attachment
+    and_i_select_a_file
+    and_i_save_a_file_attachment
```

Comment:
Minor: I think this should be a "the" here, as we're talking about a specific attachment at this point. I wonder if there's a way we could combine some of these steps, as some of them aren't doing very much and just make the file/scenario longer - it's more noticeable now there are two scenarios.

Reply:
This step no longer exists.